### DISCUSSION TITLE
Error: no module adventure_colussus

### DISCUSSION THREAD

**Discussion by robotlightsyou** [Slow-Scallion4183/Adventure-Colussus#42] [Q&A]
Hi team, 

I tried running the game for the first time in a while and I got an error that there was no module adventure_colussus, has anyone else run into this? I'm trying to run it by doing "python3 \_\_main__.py" at the command line, is that how it works when you set up the files like this?   

Using a fresh pull of the dev branch.

**Comment by Slow-Scallion4183** [comment]
ill give it a try and ill comment my results

edit: (i am also getting this problem) this is really confusing. considering that none of the files have imported 'adventure_collussus' im not sure why we are getting the error. another error that i am getting is that it cant find ```pydantic``` even though i have it installed. confusion 100

**Reply by ghost** [reply]
ill check the code. if i find anything fishy ill try to fix it.

**Reply by robotlightsyou** [reply]
I get an error on pydantic and pytest, but they don't stop me from running the test module. When I try to run \_\_main.py__ it actually stops execution, so I think there's something different going on.

I'm gonna go ahead and open an issue on this and see if any of the others don't know what's going on, I'm out of my depth.

**Reply by robotlightsyou** [reply]
SkepticalPotato2k: just wanted to give you a quick heads up this was user error, not a code issue. Check out the comments in the [issues section(#43)](https://github.com/Slow-Scallion4183/Adventure-Colussus/issues/43) for the details.
